Snake/BusinessLayer/GameSnake.cs

```diff
@@ -7,27 +7,30 @@
 namespace BusinessLayer
 {
     public class GameSnake
-    {
+    {
         enum Direction {right, down, left, up};
         Direction directionFacing;
         private int snakeLength;
         private List<Point> currentPosition;
 
         //Method to create new snake
         public List<Point> createFirstSnake(int width, int height, int newLength)
```

Comment:
Name should be CreateFirstSnake(.....
Why is the name of the method CreateFirstSnake ?
Would you create more snake inside this instance? if so you have problem with them overriding each others directions as this is single private variable, is that the desired behavior?